moveit_core/CMakeLists.txt

```diff
@@ -173,16 +118,70 @@ add_subdirectory(collision_detection_fcl)
 
 
 install(
-  TARGETS ${THIS_PACKAGE_LIBRARIES}
-  EXPORT ${PROJECT_NAME}Targets
+  TARGETS
+    moveit_butterworth_filter
+    moveit_collision_distance_field
+    moveit_collision_detection
+    moveit_collision_detection_fcl
+    moveit_collision_detection_bullet
+    moveit_dynamics_solver
+    moveit_constraint_samplers
+    moveit_distance_field
+    moveit_exceptions
+    moveit_kinematics_base
+    moveit_kinematic_constraints
+    moveit_kinematics_metrics
+    moveit_macros
+    moveit_planning_interface
+    moveit_planning_scene
+    moveit_planning_request_adapter
+    # TODO: Port python bindings
+    # moveit_python_tools
```

Comment:
Do this TODO need to stick around?

Reply:
There is already the TODO above on line 91 for this, so I'm going to remove this comment.